src/Analyzers/CSharp/Tests/AddBraces/AddBracesTests.cs

```diff
@@ -674,6 +668,8 @@ static void Main()
         [InlineData((int)PreferBracesPreference.Always, true)]
         public async Task FireForIfNestedInElseWithoutBracesWithMultilineContext5(int bracesPreference, bool expectDiagnostic)
         {
+            var x = PreferBracesPreference.WhenMultiline;
```

Comment:
Unused?

Reply:
Yes, I added that to take the screenshot above and forgot to revert.